src/components/pages/room.js

```diff
@@ -42,6 +42,6 @@ export default function Room({ PageTemplate, initialState }) {
 Room.propTypes = {
   PageTemplate: PropTypes.func.isRequired,
   initialState: PropTypes.shape({
-    roomDetails: roomDetailsShape
+    roomDetails: roomShape
```

Comment:
Hmm, the room models have little data, I don't think there is more on the room details page to be added, right?
I would keep it simple as long as we don't deal with big data models.

Reply:
I think there will be more coming from the server, we won't save it in the room collection, but there will be other stuff mapped on to the room details which is aggregated from other collections and used by the page.
But we will change the code, no problem. This is why I made this shape for the details page rather than a simple room shape (lazy loading when it comes to loading rooms without the details).

Reply:
We don't usually aggregate stuff, we pass it as separate objects. I think the only thing we fill in is probably the user 🤔 
Yeah I would change it if and when we need to, so far I would really keep it simple with this data volume and considering so far both pages use the same data.